src/profile-logic/tracks.js

```diff
@@ -394,36 +394,56 @@ export function getLocalTrackName(
   }
 }
 
+// Any thread with less than 1% non-idle time will be hidden.
+const PERCENTAGE_ACTIVE_SAMPLES = 0.01;
+
 /**
  * Determine if a thread is idle, so that it can be hidden. It is really annoying for an
- * end user to load a profile full of empty and idle threads.
+ * end user to load a profile full of empty and idle threads. This function goes through
+ * all of the samples in the thread, and sees if some large percentage of them are idle.
```

Comment:
question: does that mean that older profiles that don't have this category won't have the logic applied to them?
Are we OK with this?

Reply:
There are two cases where this code is run:

1. New profile captures.
2. Profiles viewed with no existing URL encoded viewing information.

For case 1. this will probably affect people profiling release, which [Bug 1462784](https://bugzilla.mozilla.org/show_bug.cgi?id=1462784) rides the trains here in a few days, so I don't think this is really an issue.

For case 2. This could potentially affect looking at older Talos profiles, which seems like a smaller percentage of viewers, but most profiles should have URL information already associated with them.

I am OK with this, as it makes the code we maintain simpler, with only a few end-users potentially affected. What do you think @julienw?

Reply:
got it, I didn't think (I should think more :) ) that most profiles actually have the information in the URL. So good for me :)